README.md

```diff
@@ -1,83 +1,124 @@
 
 > If you fail to upload you SSH Key you will get **permission denied** error during the deployment.
 
-## Getting Started
+### Getting Started
 
 After registration and uploading of your SSH Key you can proceed to deploy it into our system. At this stage we suggest you to:
-* [Create a team](http://docs.elastic.io/page/team-management) to work on your new component. This is not required but will be automatically created using random naming by our system so we suggest you name your team accordingly.
-* [Create a repository](http://docs.elastic.io/page/repository-management) where your new component is going to *reside* inside the team that you have just created.
+*   [Create a team](https://support.elastic.io/support/solutions/articles/14000048250-how-to-create-and-manage-a-team) to work on your new component. This is not required but will be automatically created using random naming by our system so we suggest you name your team accordingly.
+*   [Create a repository](https://support.elastic.io/support/solutions/articles/14000038797-create-and-manage-your-repositories) inside the team to deploy your new component code.
 
-## Examples of Node.js components
+### Examples of Node.js components
 
-We have numerous components build on Node.js apart from above mentioned [hello-world-nodejs](https://github.com/elasticio/hello-world-nodejs). Here are some  other community component examples:
+Here is a list of components build on Node.js:
 
-* [code-component](https://github.com/elasticio/code-component) to run pieces of synchronous JavaScript inside your integration flow,
-* [webhook-component](https://github.com/elasticio/webhook-component) to send and receive WebHooks on elastic.io platform,
-* [csv-component](https://github.com/elasticio/csv-component) to work with CSV files in your integration flow,
-* [sugarcrm-component](https://github.com/elasticio/sugarcrm-component) to use Sugar CRM in your integrations
-* and many more ...
+*   [petstore-component-nodejs](https://github.com/elasticio/petstore-component-nodejs) to build your first component
+*   [code-component](https://github.com/elasticio/code-component) to run pieces of synchronous JavaScript inside your integration flow,
+*   [webhook-component](https://github.com/elasticio/webhook-component) to send and receive WebHooks on elastic.io platform,
+*   [csv-component](https://github.com/elasticio/csv-component) to work with CSV files in your integration flow,
+*   [sugarcrm-component](https://github.com/elasticio/sugarcrm-component) to use Sugar CRM in your integrations
 
 [travis-image]: https://travis-ci.org/elasticio/sailor-nodejs.svg?branch=master
 [travis-url]: https://travis-ci.org/elasticio/sailor-nodejs
 [daviddm-image]: https://david-dm.org/elasticio/sailor-nodejs.svg?theme=shields.io
 [daviddm-url]: https://david-dm.org/elasticio/sailor-nodejs
 
+
+
 ## Sailor hooks
 
+### Init hook
+
+```javascript
+/**
+* cfg - config object with credentials and other stuff like datasamples
```

Comment:
I would add a notice like:
`This is the same config as the one passed to `processMessage` method of the trigger or action.`